todoman/formatters.py

```diff
@@ -82,7 +82,7 @@ def detailed(self, todo):
             extra_rows += self._columnize('Description', todo.description)
         if todo.location:
             extra_rows += self._columnize('Location', todo.location)
-
+        
```

Comment:
Please remove the newly-added empty spaces here. Spaces at the end of the line will make style checks fail.